styles/styles.css

```
@@ -165,6 +211,23 @@ main {
     margin-right: auto
 }
 
+
+.advantages-photo-first {
+    flex-grow: 0;
+    flex: none;
+}
+
+.advantages-photo-secound {
+    flex-grow: 0;
+    flex: none;
+}
```

```suggestion
.advantages-photo-secound {
}
```

Аналогично. 
В этих свойствах нет смысла.